avocado/utils/astring.py

```diff
@@ -324,7 +324,6 @@ def to_text(data, encoding=None):
     """
     if is_bytes(data):
         if encoding is None:
-            return data.decode()
-        else:
-            return data.decode(encoding)
+            encoding = ENCODING
```

Comment:
Yep, I think this is OK.  The docstring is missing (my bad), but it shouldn't.  Now with this change, it seems even more necessary.  Would you please add it?